ext/cfx-ui/src/assets/languages/locale-de.json

```
@@ -44,22 +51,35 @@
     "#Settings_MenuAudioDesc": "Aktiviere Hintergrundmusik während Inaktivität im Hauptmenü",
     "#Settings_GameStreamProgress": "Streaming-Fortschritt",
     "#Settings_GameStreamProgressDesc": "Zeigt den Fortschritt beim Herunterladen von im Spiel gestreamten Inhalten an. (BETA™)",
-    "#DirectConnect_TimedOut": "Zeitüberschreitung der Serveranfrage.",
+    "#Settings_QueriesPerMinute": "Serversuche: Maximale Pings / Minute",
+    "#Settings_CustomEmoji": "Eigenes FiveM Wasserzeichen",
+    "#Settings_CustomEmojiDesc": "Ein Emoji für das Wasserzeichen im Spiel.",
+    "#Settings_CustomEmojiUpsell": "Nicht vefügbar! Spende $5/Monat oder mehr, um einen offiziellen FiveM-Patreon Account zu bekommen und Dir Dein eigenes Emoji als Wasserzeichen zu setzen.",
```

It is not an "Official FiveM Patreon Account" that you receive, but rather the linking of Patreon with the FiveM account. The translation suggests that there are actual "premium accounts" available for purchase.

```suggestion
    "#Settings_CustomEmojiUpsell": "Nicht verfügbar! Unterstütze FiveM auf Patreon mit $5/Monat oder mehr und verknüpfe dein FiveM-Konto, um ein individuelles Emoji als FiveM-Wasserzeichen festzulegen..",
```